src/main/java/geekbrains/java2/ht1b/Main.java

```diff
@@ -54,9 +63,9 @@ public static void main( String[] args ){
 
     //получаем значения в ячейках на доске
     private static String getValue(String value){
-        if(value == "1"){
+        if(value == "1") {
```

Comment:
\[Checkstyle\] ERROR: Literal Strings should be compared using equals\(\), not '=='\.